JitsiConference.js

```diff
@@ -1561,7 +1565,7 @@ JitsiConference.prototype._updateFeatures = function(participant) {
             participant._supportsDTMF = features.has('urn:xmpp:jingle:dtmf:0');
             this.updateDTMFSupport();
 
-            if (features.has('http://jitsi.org/protocol/jigasi')) {
+            if (features.has(JIGASI_XMLNS)) {
```

Comment:
This is no longer used as a XMLNS, perhaps naming it `JIGASI_FEATURE` is more appropriate